src/auth/auth_backend.ts

```diff
@@ -37,6 +37,7 @@ export enum AuthMethods {
 export interface AuthConfiguration {
   method?: AuthMethods;
   provider?: AuthProviders;
+  scope?: string[];
```

Comment:
fyi - this doesn't look like part of the $exists impl. This may not generate sufficient release notes from the commit message.